src/lib.rs

```diff
@@ -220,6 +244,29 @@ impl<T: Translation + Clone> Mapping<T> {
                     ttbrval = in(reg) self.previous_ttbr.unwrap(),
                     options(preserves_flags),
                 ),
+                (ExceptionLevel::El2, VaRange::Lower) => asm!(
+                    "msr   ttbr0_el2, {ttbrval}",
+                    "isb",
+                    "tlbi  aside1, {asid}",
```

Comment:
If this is going to be used for non-VHE EL2, this should be `tlbi alle2`

Reply:
I've made this panic instead for now, as from what you were saying before it sounds like we don't have a safe way to disable page table in non-VHE EL2 or EL3.